tests/regression.py

```diff
@@ -751,14 +762,20 @@ class TestDBAccessConninfo(BaseUpdateRunner, unittest.TestCase,
     extra_params = ['--slim', '-d', 'dbname=' + CONFIG['test_database']]
 
 class TestDBAccessConninfoLong(BaseUpdateRunner, unittest.TestCase,
-                           PgsqlBaseTests):
+                               PgsqlBaseTests):
     extra_params = ['--slim', '--database', 'dbname=' + CONFIG['test_database']]
 
 class TestDBAccessURIPostgresql(BaseUpdateRunner, unittest.TestCase,
-                           PgsqlBaseTests):
+                                PgsqlBaseTests):
     extra_params = ['--slim', '-d', 'postgresql:///' + CONFIG['test_database']]
 
 class TestDBAccessURIPostgres(BaseUpdateRunner, unittest.TestCase,
-                           PgsqlBaseTests):
+                              PgsqlBaseTests):
     extra_params = ['--slim', '-d', 'postgres:///' + CONFIG['test_database']]
 
+# Schema tests
+
+class TestDBOutputSchema(BaseUpdateRunnerWithOutputSchema, unittest.TestCase,
+                         PgsqlBaseTests):
+    extra_params = ['--slim', '--output-pgsql-schema=osm']
```

Comment:
I would expect that the `PgsqlBaseTests` do not work here because the tests use table names without a schema. Am I missing something?

Reply:
The test was accessing tables in the public namespace left over from earlier tests. Fixed now.